test/model/message_list_test.dart

```diff
@@ -2742,6 +2742,53 @@ void main() {
     checkNotifiedOnce();
   }));
 
+  group('one message per block?', () {
+    final channelId = 1;
+    final topic = 'some topic';
+    void doTest({required Narrow narrow, required bool expected}) {
+      test('$narrow: ${expected ? 'yes' : 'no'}', () => awaitFakeAsync((async) async {
+        final sender = eg.user();
+        final channel = eg.stream(streamId: 1);
```

Comment:
nit: implicitly relying on these stream/channel IDs being equal, IIUC; instead mention the variable